### PR TITLE
Adding visitSequence param to the long endpoint

### PR DESCRIPTION
**Pull request by chStaiger**
- visitSequence added, documentation in Documentation_endpoints.md
- clean up of Example of long endpoint in openapi.yaml
- method added
- formula added

### CODE REVIEW

**Comment by chStaiger** [comment]
@RoelBrouwer I finished the last parameters apart from the dots. This PR is now safe to review.

File: webmice_functions.R [hunk 1]
```diff
@@ -53,7 +53,7 @@ imp_result_long_fmt <- function(imp) {
 #' @inheritParams mice::mice
 impute <- function(data, maxit, m, seed, 
                    blocks, parcel, predictorMatrix, ignore, where,
-                   visitSequence) {
+                   visitSequence, method) {
```
Comment (RoelBrouwer) [inline webmice_functions.R:56]: Should the formula argument not be in here as well?

File: webmice_functions.R [hunk 2]
```diff
@@ -62,7 +62,7 @@ impute <- function(data, maxit, m, seed,
                   predictorMatrix = predictorMatrix, 
                   blocks = blocks, parcel = parcel,
                   ignore = ignore, where = where,
-                  visitSequence = visitSequence
+                  visitSequence = visitSequence, method = method
```
Comment (RoelBrouwer) [inline webmice_functions.R:65]: Should the formula argument not be in here as well?

File: webmice_functions.R [hunk 3]
```diff
@@ -161,7 +193,8 @@ call_mice <- function(params) {
                 predictorMatrix = pm,
                 ignore = ign,
                 where = whr, 
-                visitSequence = visitSeq
+                visitSequence = visitSeq,
+                method = method
```
Comment (RoelBrouwer) [inline webmice_functions.R:197]: Should the formula argument not be in here as well?

**Pull request merged**